client/cypress/integration/dashboard/dashboard_spec.js

```diff
@@ -256,15 +256,15 @@ describe('Dashboard', () => {
           });
 
         const newContent = '[edited]';
-        cy.get('edit-text-box')
+        cy.get('.textbox-dialog')
```

Comment:
You can add `wrapProps={{ 'data-test': 'TextboxDialog' }}` to the dialog and then use it in tests (https://github.com/getredash/redash/pull/3493/files#diff-0c11d795d0fc2f8315e6f3a946848ea7R193)

Reply:
Oooh I'll do that.